.github/workflows/super-linter-non-slim.yml

```diff
@@ -65,3 +65,4 @@ jobs:
           VALIDATE_KUBERNETES_KUBEVAL: false
           YAML_CONFIG_FILE: yaml/.yaml-lint.yml
           TERRAFORM_TFLINT_CONFIG_FILE: terraform/.tflint.hcl
+          SQLFLUFF_CONFIG_FILE: sqlfluff/.sqlfluff-lint
```

Comment:
@wiebeck I missed to include it because of 
![Screenshot 2023-05-31 at 13 16 37](https://github.com/riege/code-quality/assets/8565560/201636f9-2b0d-4da9-858c-f249a4d5edc1)